src/Tribe/Commerce/PayPal/Oversell/Request.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * Class Tribe__Tickets__Commerce__PayPal__Oversell__Request
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Commerce__PayPal__Oversell__Request {
+
+	/**
+	 * @var string
+	 */
+	public static $oversell_action = 'oversell';
+
+	/**
+	 * @var string
+	 */
+	protected $policy;
+
+	/**
+	 * @var string
+	 */
+	protected $order_id;
+
+	/**
+	 * Conditionally handles an oversell request.
+	 *
+	 * @since TBD
+	 */
+	public function handle() {
+		if ( ! isset( $_GET['tpp_action'], $_GET['tpp_policy'], $_GET['tpp_order_id'], $_GET['tpp_slug'] ) ) {
+			return;
+		}
+
+		if ( self::$oversell_action !== $_GET['tpp_action'] ) {
+			return;
+		}
+
+		$this->policy   = $_GET['tpp_policy'];
+		$this->order_id = $_GET['tpp_order_id'];
```

Comment:
Why no go with `Tribe__Array__Utils`, here and on the checks above.

Reply:
I've checked on line 31 at that point and know the values are there .

Reply:
Sounds good